addons/mail/static/src/messaging/component/composer/composer_tests.js

```diff
@@ -197,22 +208,33 @@ QUnit.skip('display partner mention suggestions on typing "@"', async function (
     document.querySelector(`.o_ComposerTextInput_textarea`)
         .dispatchEvent(new window.KeyboardEvent('keyup'));
     await afterNextRender();
```

Comment:
While the previous syntax is still supported, it can lead to concurrency issues, so the new one is preferable. We might need to do a pass on the whole code, so it's fine if you don't update this one since it is an existing one and you have no diff on the line, but since I saw it, I might as well note it.

```suggestion
    await afterNextRender(() => {
        document.querySelector(`.o_ComposerTextInput_textarea`)
            .dispatchEvent(new window.KeyboardEvent('keyup'));
    });
```

I don't remember if you already got the explanation for this one, so here it is
`afterNextRender` has been updated to take as parameter the action that is supposed to trigger the render.
This allows the "render listener" to be bound before the action takes place, to ensure it will always catch the effect of the action.

Reply:
Ok